game/paralysis.cc

```diff
@@ -0,0 +1,11 @@
+#include "paralysis.h"
+
+Paralysis::Paralysis(Hackmon & h) : Mobility(h), rng{RandomGenerator(1,2)} {}
+
+bool Paralysis::canMoveOverride() {
+  if(rng.getRandom() >= 7) {
```

Comment:
`rng` has maximum value 2, so it will never be >= 7

Reply:
you got me there, i meant to make it 10 haha